source/common/http/conn_pool_base.h

```diff
@@ -105,9 +105,9 @@ class HttpConnPoolImplBase : public Envoy::ConnectionPool::ConnPoolImplBase,
 // An implementation of Envoy::ConnectionPool::ActiveClient for HTTP/1.1 and HTTP/2
 class ActiveClient : public Envoy::ConnectionPool::ActiveClient {
 public:
-  ActiveClient(HttpConnPoolImplBase& parent, uint64_t lifetime_stream_limit,
-               uint64_t effective_concurrent_stream_limit,
-               uint64_t configured_concurrent_stream_limit,
+  ActiveClient(HttpConnPoolImplBase& parent, uint32_t lifetime_stream_limit,
```

Comment:
What does 'lifetime' mean here? I'm just wondering if we are counting a potentially huge number of events over the lifetime of a server.

Reply:
It is the upstream's connection's life time to the backends, and it could be large, but it is not the lifetime of a server.